defs/scenarios/openstack/nova/bugs.yaml

```diff
@@ -25,6 +30,14 @@ checks:
     expr: '.+attaching network adapter failed.: libvirt.libvirtError: Requested operation is not valid: PCI device'
     hint: 'libvirtError'
 conclusions:
+  lp1904580:
+    decision: has_1904580
+    raises:
+      type: LaunchpadBug
+      bug-id: 1904580
+      message: >-
+        known nova compute package upgrade bug which affects private ssh key
+        permissions. Change private key file permission to 0600 to fix the issue.
```

Comment:
Would it make sense/be possible to include the path of the file in your message?

Reply:
done